docs/adr/0010-tosca-yaml-deserialisation-using-snakeyaml.md

```diff
@@ -0,0 +1,45 @@
+# TOSCA YAML deserialization using SnakeYAML
+
+The TOSCA YAML files have to be read into a Java model (deserialized) and written from the Java model into files (serialized).
+
+## Considered Alternatives
+
+* [SnakeYAML](https://bitbucket.org/asomov/snakeyaml) writing into intermediate model consisting of default Java types
+* SnakeYAML writing into final Java model
+* Manual reading
+
+## Conclusion
```

Comment:
Could you rename that to "Decision Outcome"

We renamed the heading - see https://github.com/eclipse/winery/blob/master/docs/adr/template.md

https://github.com/eclipse/winery/blob/master/docs/adr/template.md